workers/unity/Assets/Playground/Config/PrefabConfig.cs

```diff
@@ -1,29 +0,0 @@
-using System.Collections.Generic;
-
-namespace Playground
-{
-    public static class PrefabConfig
```

Comment:
Please update `https://github.com/spatialos/UnityGDK/blob/master/docs/content/entity-checkout-process.md` since we are invalidating existing documentation here.